dmapiclient/data.py

```diff
@@ -563,15 +568,16 @@ def get_brief(self, brief_id):
         return self._get(
             "/briefs/{}".format(brief_id))
 
-    def find_briefs(self, user_id=None, status=None, framework=None, lot=None, page=None, human=None):
+    def find_briefs(self, user_id=None, status=None, framework=None, lot=None, page=None, human=None, with_users=None):
         return self._get(
             "/briefs",
             params={"user_id": user_id,
                     "framework": framework,
                     "lot": lot,
                     "status": status,
                     "page": page,
-                    "human": human
+                    "human": human,
+                    "with_users": with_users
```

Comment:
@pcraig3 Have fixed the user_id thing in the api [here](https://github.com/alphagov/digitalmarketplace-api/pull/431/files#diff-b06835e202eca80d421684ffb7ba77b2R143). Just added the with users to the serialize method.

Also, having just spoken to you about the `'True'` v `'true'` thing, I'm going to handle it in the api by lowering whatever comes through so either should work.